src/main/java/ru/yandex/practicum/filmorate/storage/film/DbFilmStorage.java

```diff
@@ -28,30 +29,28 @@
 public class DbFilmStorage implements FilmStorage {
     private final JdbcTemplate jdbcTemplate;
     private final DbUserStorage dbUserStorage;
+    private final DbDirectorStorage dbDirectorStorage;
```

Comment:
Лучше обозначать интерфейс: 
```suggestion
    private final DirectorStorage dbDirectorStorage;
```
Так программа будет более гибкой, и если изменится реализация, ничего не сломается
Предлагаю поправить еще и 31 строку

Reply:
Исправил оба